bin/regenerate-travis.sh

```diff
@@ -1,5 +0,0 @@
-#!/bin/sh -eu
-
-# .travis.yml is generated using the 'haskell-ci' available on Hackage
-
-haskell-ci regenerate --copy-fields=all
```

Comment:
Is `haskell-ci --copy-field=all cabal.project` an alternative? If yes, I think it makes sense to remove this (as you did).

Reply:
Indeed. In general, once the correct incantation is recorded in the comment at the top of the `.travis.yml`, `haskell-ci regenerate` should be sufficient.

Reply:
Good to know :+1: Yes, as per @phadej in https://github.com/haskell-CI/haskell-ci/issues/377#issuecomment-609384926.